corehq/apps/data_analytics/esaccessors.py

```diff
@@ -52,7 +52,6 @@ def active_mobile_users(domain, start, end, *args):
                   .domain(domain.name)
                   .user_aggregation()
                   .submitted(gte=start, lt=end)
-                  .user_id(user_ids)
```

Comment:
I removed the user_id filtering from these ES queries since it doesn't make sense to me. I think the only downside is that  we may include the `system` user.

Reply:
@calellowitz  do you remember what your intention might have been when you added this line as part of adding this es-accessor 3 and a half years ago? 😂https://github.com/dimagi/commcare-hq/commit/7e4d17162c2f0654f7c0fe791573877b3f8e0121

@snopoke as far as I can tell your take on this makes sense